src/i18n/es.js

```diff
@@ -92,31 +89,19 @@ export default {
       <p>
         Activa el periodo de prueba y accede a Reportes detallados sobre el comportamiento de los
         usuarios en tu Sitio Web o E-commerce. Descubre cuáles son las páginas más visitadas, cuántos
-        visitantes poseen un Email que Doppler ha identificado y cuántos no.
+        visitantes poseen un Email que Doppler ha identificado y cuántos no. ¿Necesitas ayuda?
+        <a target="_blank" href="${urlHelpAdvancedReports}" ${patchForBlank}>HELP</a>.
```

Comment:
Hi @carodipietro, I have added this text in Spanish on the page of trial activation.

I think that we have missed it before. 

![image](https://user-images.githubusercontent.com/1157864/57876225-3d3df000-77eb-11e9-9900-8fdd003bb306.png)

What do you think?

Reply:
Mergeo, si está mal, me avisan y revierto.